source/includes/api/v3/_households.md

```diff
@@ -245,7 +245,7 @@ Authorization: Bearer {access_token}
 
 ### HTTP Request
 
-`POST /households`
+`POST /v2/households`
```

Comment:
I believe that all other `HTTP Request` sections have the full domain `https://api.beta.polisapp.com/...`